src/components/mods/modsTable.tsx

```diff
@@ -16,16 +16,35 @@ import type { ModWithInfo } from "~/components/mods/types";
 
 const PAGE_SIZES = [5, 10, 15, 20, 25, 50, 100, 250, 500, 1000];
 const DEFAULT_PAGE_SIZE_INDEX = 1;
-
+const TAB_COLORS = ["#a7a7a7", "#b7c3dc", "#92a6c5", "#527da2", "#263972"];
```

Comment:
I used the colors in the mockup - I think they need to be changed for better contrast.

Reply:
Agreed. Please tweak the colors.